std/datetime.d

```diff
@@ -29161,6 +29191,72 @@ private:
         _hasDST = hasDST;
     }
 
+    // Android concatenates the usual timezone directories into a single file,
+    // tzdata, along with an index to jump to each timezone's offset.  In older
+    // versions of Android, the index was stored in a separate file, zoneinfo.idx,
+    // whereas now it's stored at the beginning of tzdata.
+    version(Android)
+    {
+        // Keep track of whether there's a separate index, zoneinfo.idx
+        static bool separate_index;
```

Comment:
Looking into making this all immutable at module scope and initialized by a shared module constructor.